src/aat/resources/features/F-037/F-037.feature

```diff
@@ -66,3 +66,24 @@ Feature: F-037: Submit event for an existing case (V2)
     Then a negative response is received
     And the response [contains a HTTP 404 'Not Found']
     And the response has all the details as expected
+
+  @S-577
+  Scenario: must update successfully all and only the respective fields with update access for a Case Worker and a Solicitor
+    Given a user with [an active Case Worker profile in CCD with a specific variation of ACLs on a case type]
+    And a user with [an active solicitor profile with another specific variation of ACLs on the same case type]
+    And a successful call [to create a token for case creation] as in [Befta_Jurisdiction2_Default_Token_Creation_Data_For_Case_Creation]
+    And another successful call [by a privileged user with full ACL to create a case of this case type] as in [Befta_Jurisdiction2_Default_Full_Case_Creation_Data]
+    And another successful call [to get an update event token for the case just created] as in [S-577-Prerequisite_Solicitor_2_Token_For_Update_Case]
+    When a request is prepared with appropriate values
+    And the request [is prepared on behalf of the user creating the case above]
```

Comment:
could describe what the request is doing, eg. `And the request [prompts an update to DocumentField2, made by the privileged user who just created the case]`

Reply:
Agreed and updated.